playwright.local.config.ts

```diff
@@ -0,0 +1,26 @@
+// playwright.local.config.js
```

Comment:
Based on https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests and https://playwright.dev/docs/test-cli, in order to have different configs (to allow local development), require two different config files.